docs/requirement-sets/common/office-add-in-requirement-sets.md

```diff
@@ -175,7 +172,7 @@ See [Add-in command requirement sets](add-in-commands-requirement-sets.md).
 
 | Minimum Office application support | Methods in set |
```

Comment:
Do we even need to include this requirement set? It's not an API, so it doesn't fall under "public surface area".